src/styles/modal.css

```
@@ -5,83 +5,300 @@ body.modal-open {
+  font-size: 1.2rem;
+  font-weight: 700;
+  color: var(--color-bluegray-10);
 }
 
 .detail {
-  max-height: 430px;
+  max-height: min(48vh, 420px);
   overflow-y: auto;
+  padding-right: 8px;
+}
+@keyframes slideUp {
```

Stylelint (17.6.0)

[error] Expected keyframe name "slideUp" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)